src/store/user/actions.js

```diff
@@ -17,17 +17,52 @@ export const loginError = () => ({
     type: LOGIN_ERROR,
 });
 
-export const login = (login, password) => async (dispatch) => {
+export const login = (login, password, onSuccess) => async (dispatch) => {
     dispatch(loginPending());
-    const { status } = await authLogin(login, password);
+    const response = await authLogin(login, password);
+    const { status } = response;
+    const data = await response.json();
     if (status === 200 || status === 204) {
         dispatch(
             loginSuccess({
                 login,
                 password,
             })
         );
+        localStorage.setItem('authToken', data?.authToken);
+        localStorage.setItem('refreshToken', data?.refreshToken);
+        onSuccess?.();
     } else {
         dispatch(loginError());
     }
 };
+
+export const SIGNUP_PENDING = 'SIGNUP_PENDING';
+export const SIGNUP_SUCCESS = 'SIGNUP_SUCCESS';
+export const SIGNUP_ERROR = 'SIGNUP_ERROR';
+
+export const signupPending = () => ({
+    type: SIGNUP_PENDING,
+});
+
+export const signupSuccess = (userData) => ({
```

Comment:
please split actions and reducers

Reply:
Actions and reducers are already in the separate files. Did you mean action types or thunks?